server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/InheritanceActionTest.java

```diff
@@ -209,11 +206,18 @@ public void stat_for_all_profiles() {
     dbClient.ruleDao().insert(dbSession, rule);
     dbSession.commit();
 
+    // FIXME QProfileServices checks for wrong permission, currently
```

Comment:
This change will however be part of #1822 , since it is actually not related to the changes in this PR. (I just found the bug during the development of this feature). I'll only move the FIXME to the production code, so that SonarQube will see it.